charts/ratify/templates/assign.yaml

```diff
@@ -0,0 +1,50 @@
+apiVersion: mutations.gatekeeper.sh/v1beta1
```

Comment:
The latest API version is `mutations.gatekeeper.sh/v1` (xref: https://open-policy-agent.github.io/gatekeeper/website/docs/mutation/)

Reply:
cc @sozercan to comment on the supported releases and api version

Reply:
updated to use api version based on gatekeeper version selected